test/test_opsum_to_ttn.jl

```diff
@@ -125,19 +141,78 @@ using Test
     end
   end
 
+  @testset "OpSum to TTN Fermions" begin
+    # small comb tree
+    auto_fermion_enabled = ITensors.using_auto_fermion()
+    if !auto_fermion_enabled
+      ITensors.enable_auto_fermion()
+    end
+    tooth_lengths = fill(2, 3)
+    c = named_comb_tree(tooth_lengths)
+    is = siteinds("Fermion", c; conserve_nf=true)
+
+    # test with next-nearest neighbor tight-binding model
+    t = 1.0
+    tp = 0.4
+    U = 0.0
+    h = 0.5
+    H = ITensorNetworks.tight_binding(c; t=t, tp=tp, h=h)
```

Comment:
```suggestion
    H = ITensorNetworks.tight_binding(c; t, tp, h)
```
and same thing elsewhere.